src/test/java/com/epam/lowcost/service/implementations/PlaneServiceImplTest.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.lowcost.service.implementations;
+
+import static org.mockito.Mockito.*;
+import static org.junit.Assert.*;
+
+import com.epam.lowcost.DAO.interfaces.PlaneDAO;
+import com.epam.lowcost.model.Plane;
+import com.epam.lowcost.service.interfaces.PlaneService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PlaneServiceImplTest {
+
+    @Mock
+    private PlaneDAO planeDAO;
+
+    @InjectMocks
+    private PlaneService planeService = new PlaneServiceImpl();
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void getAllPlanes() {
+        List<Plane> expectedList = new ArrayList<>();
+
+        when(planeDAO.getAllPlanes()).thenReturn(new ArrayList<Plane>());
+
+        assertEquals(expectedList, planeService.getAllPlanes());
+    }
+
+    @Test
+    public void getById_positiveIntegetNumberInRangeAsParam_PlaneEntetyReturned() {
+        Plane expectedPlane = new Plane(1, "Airbus A380", 25, 500, false);
```

Comment:
Because you haven't any business logic into service layer, I think it would be enough to verify a calling DAO method with right parameters (as it was done in 'addPlane' test, f.e.).